Symfony/CS/AbstractFunctionReferenceFixer.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS;
+
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @internal base class for function reference fixers
```

Comment:
Not sure if it's legal. I got used to annotate this way:

```php
/**
 * @internal used inside \namespace only
 * @deprecated use \class::method instead
 */
```
Maybe incorrect, but explains why symbol is deprecated/intenal/inspection suppressed